src/components/ClientCard/ClientCard.jsx

```diff
@@ -20,24 +20,20 @@ const ClientCard = props => (
   </li>
 );
 
-const mapStateToProps = state => ({
-
-});
-
 
 const mapDispatchToProps = dispatch => ({
-  setActiveChat: chat => dispatch(setActiveChat(chat)),
+  setActiveChat: chatId => dispatch(setActiveChat(chatId)),
 });
 
 
 export default connect(
-  mapStateToProps,
+  () => ({}),
```

Comment:
Interesting. I tried it with `null` and it seemed to work.

I feel like maintaining the function expression for `mapStatetoProps` makes sense for now anyway. The fact that this component uses dispatch ... well it might eventually have some state too. So we might as well keep the readability in there.